sql/moz-fx-data-shared-prod/analysis/bqetl_default_task_v1/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+friendly_name: Defult Task for DAG bqetl_default
+description: |-
+  Default task to generate the default DAG.
+  See https://github.com/mozilla/bigquery-etl/issues/3650.
```

Comment:
```suggestion
friendly_name: Default Task for DAG bqetl_default
description: |-
  Placeholder task to generate the default DAG.
  See https://github.com/mozilla/bigquery-etl/issues/3650.
  This query returns no rows and is just used to ensure that the bqetl_default DAG
  has at least one task, otherwise the DAG would be invalid.
```